.github/workflows/pullrequest-trust-helper.yaml

```diff
@@ -0,0 +1,16 @@
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+jobs:
+  pullrequest-trusted-helper:
+    permissions:
+      pull-requests: write
+    secrets: inherit # access to `GitHub-Actions`-App is needed to read teams
+    uses: gardener/cc-utils/.github/workflows/pullrequest-trust-helper.yaml@master
+    with:
+      trusted-teams: 'core,mcm-maintainers,gardener-extension-provider-openstack-maintainers'
```

Comment:
Do we need core to be mentioned here since they're not the team reviewing PRs on this repo?